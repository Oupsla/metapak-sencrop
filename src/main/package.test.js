"use strict";

const packageTransformer = require("./package");

describe("Main", () => {
  describe("Package transformer", () => {
    test("should work with an empty package.json", () => {
      expect(
        packageTransformer({
          metapak: {
            data: {
              files: "lol.js"
            }
          }
        })
      ).toMatchSnapshot();
    });
  });
});
