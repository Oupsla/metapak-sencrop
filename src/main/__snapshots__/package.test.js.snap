// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Package transformer should work with an empty package.json 1`] = `
Object {
  "config": Object {
    "commitizen": Object {
      "path": "./node_modules/cz-conventional-changelog",
    },
  },
  "dependencies": Object {},
  "devDependencies": Object {
    "commitizen": "^2.9.6",
    "conventional-changelog-cli": "^1.2.0",
    "cz-conventional-changelog": "^2.0.0",
  },
  "engines": Object {
    "node": ">=6.10.0",
  },
  "license": "SEE LICENSE IN LICENSE.md",
  "metapak": Object {
    "data": Object {
      "files": "lol.js",
    },
  },
  "scripts": Object {
    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
    "cli": "env NODE_ENV=\${NODE_ENV:-cli}",
    "cz": "env NODE_ENV=\${NODE_ENV:-cli} git cz",
    "lint": "echo \\"WARNING: No linter configured\\"",
    "preversion": "npm run lint",
    "version": "npm run changelog && git add CHANGELOG.md",
  },
  "version": "0.0.0",
}
`;
